source/features/delete-review-comments-faster.css

```diff
@@ -0,0 +1,3 @@
+.rgh-delete-button-form::before, .rgh-delete-button-form::after {
+	content: none !important;
+}
```

Comment:
Could not find a GH class to remove the `::before` and `::after`. (They get in the way of the flexbox layout)